.github/workflows/build-vcpkg.yml

```diff
@@ -336,6 +336,16 @@ jobs:
           cmake -S ./HPCC-Platform -B ./build ${{ matrix.cmake_config_options }}
           cmake --build ./build ${{ matrix.cmake_build_options }}
 
+      - name: WinTest
```

Comment:
@GordonSmith Does this look ok?  Any problem with directories?

Reply:
@GordonSmith @ghalliday - please ignore this PR (closing). This was meant to be a test PR against my own repo. not a change/PR for review.